rfc/0075-faster-failover-and-configuration-push.md

```diff
@@ -0,0 +1,337 @@
+is negotiated, the node might send an empty body with `NotMyVbucket` (`0x07`) status codes. The KV engine tracks the
+revision that has been sent to the SDK over the socket connection, so a response with a `NotMyVbucket` status will only
+have a body if the pushed version is older than the active configuration.
+
+The KV engine updates the pushed configuration version in the following cases:
+* Configuration sent to the SDK in response to a `GetClusterConfig` (`0xb5`) request.
+* Configuration pushed to the SDK that enabled the HELLO flag `ClustermapChangeNotification` (`0x0d`).
+
+NOTE: `DedupeNotMyVbucketClustermap` affects `ClustermapChangeNotification` and `ClustermapChangeNotificationBrief`
+features, that described below. In other words, if deduplication enabled, the cluster configuration will be announce for
+the socket connection only once.
+
+NOTE: KV engine does not inspect the body of the configuration, it only looks at `epoch`/`revision` pair to detect
+duplicates (decide if the SDK seen the update). It means, that SDK still might receive updates that do not change
+VBucket map or topology (number, order or names of the nodes). It is up to the SDK to decide if the configuration
+affects the sockets or pipeline configuration, which must be already implemented for older generation of the servers.
+
+### Enforcing Snappy Compression for Cluster Configuration Payloads
+
+[https://review.couchbase.org/c/kv_engine/+/192152]: #
+[https://review.couchbase.org/c/kv_engine/+/192316]: #
+
+The KV engine introduces a new HELLO flag called `SnappyEverywhere` with a value of `0x13`. Once this flag is
+negotiated, the node will always use the compressed version of the cluster configuration and data type flags will be set
+to `JSON | SNAPPY` (`0x03`).
+
+Note, that the meaning of the flag `SnappyEverywhere` is that SDK expects and properly handles compression for **ANY**
+operation during communication with the KV engine, this is why the flag called "SnappyEverywhere", and "SnappyConfig".
+
+### `GetClusterConfig` and Out-of-Order Execution
+
+[https://issues.couchbase.com/browse/MB-56885]: #
+
+HELLO flag `UnorderedExecution` (`0x0e`) enables Out-of-Order (OoO) execution, so that the KV engine is being allowed to
+reorder operations. [kv\_engine/docs/UnorderedExecution.md][kv-unordered-execution] provides more details on this
+feature.
+
+The `GetClusterConfig` (`0xb5`) command is explicitly marked as compatible with OoO execution, allowing it to be served
+without waiting for the completion of in-flight operations. Specifically, `GetClusterConfig` will not wait for long
+operations such as mutations with SyncDurability requirements. All current SDKs are expected to be compatible with the
+OoO execution mode, so no changes are expected.
+
+### Cluster Configuration Notification Changes
+
+Prior to server version 7.6, the KV engine had an opt-in feature to push configuration updates to SDKs. This feature
```

Comment:
Does the `ClustermapChangeNotification` feature still exist in 7.6?

Reply:
yes, it does. And the RFC has note that it should not be used in HELLO handshake.